interface/modules/zend_modules/module/PrescriptionTemplates/src/PrescriptionTemplates/Controller/PdfTemplatesController.php

```diff
@@ -35,11 +36,12 @@
  */
 class PdfTemplatesController extends PrescriptionTemplatesController
 {
-    private $renderer;
+    public $renderer;
```

Comment:
Can you explain why this was made 'public'?  If you want PrescriptionTemplatesController to have access to it, you can just make it 'protected' instead of 'public'?  Are you wanting the view to somehow have access to this property even though I'm not seeing it used anywhere?